### ISSUE TITLE
Configurable cache directory

### ISSUE THREAD

**Issue by promisedlandt** [danchoi/ri.vim#31]
Hello,

currently the cache directory is hardcoded to be the `ri_vim_cache` subdirectory of the plugin path. Would it be possible to make this configurable, so I could set it to something like `/tmp/ri_vim_cache` instead?

**Comment by wikimatze** [comment]
Sure, have you already tried it?

**Comment by promisedlandt** [comment]
Yes, I have created a branch [here](https://github.com/promisedlandt/ri.vim/tree/configurable-tmp-dir) for my personal use, but I figured someone who knows what they're doing in vimscript can do better :)

**Comment by wikimatze** [comment]
@promisedlandt looks good I'll try it out with neovim.
